test/unit/controllers/Irc_EntryController.test.js

```diff
@@ -177,23 +195,25 @@ describe('UNIT Irc_EntryController', () => {
     });
 
     after(() => {
+      clock.restore();
       global.Centres = original_centre;
     });
 
-    it('should update the centre', () =>
-	expect(global.Centres.update).to.be.calledWith(
+    it('should update the centre', () => {
```

Comment:
why did this need to have a curly brace?
if you need them you should return the `expect`